pkg/component/machinecontrollermanager/bootstrap.go

```diff
@@ -33,6 +33,7 @@ import (
 const (
 	managedResourceControlName = "machine-controller-manager"
 	clusterRoleName            = "system:machine-controller-manager-runtime"
+	unsupportedClusterRoleName = "system:machine-controller-manager-seed"
```

Comment:
Maybe add a `TODO` statement to remove this in further release once the cleanup is done.

Reply:
after v1.88 has been released